app/src/main/res/values-xxxhdpi/dimens.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="dialpad_button_size">72dp</dimen>
+    <dimen name="dialpad_button_padding">16dp</dimen>
+    <dimen name="dialpad_button_size_small">50dp</dimen>
+    <dimen name="incoming_call_arrow_size">50dp</dimen>
+    <dimen name="incoming_call_button_size">72dp</dimen>
+    <dimen name="call_notification_button_size">40dp</dimen>
+    <dimen name="incoming_call_avatar_size">120dp</dimen>
+
+    <dimen name="dialpad_text_size">30sp</dimen>
+    <dimen name="caller_name_text_size">34sp</dimen>
+    <dimen name="call_status_text_size">20sp</dimen>
+</resources>
```

Comment:
oh cmon, avoid copypasting all these files, it is a road to hell

Reply:
you mean to not duplicate the same sizes in different dimens.xml files, right? 
or
do you want to keep using a single dimension file for all screen sizes?

I increased the text sizes for sw480dp and sw720dp a little bit

Reply:
removed duplicated stuff